src/pretix/control/forms/widgets.py

```diff
@@ -77,3 +77,19 @@ def options(self, name, value, attrs=None):
 
 class Select2ItemVarQuota(Select2ItemVarQuotaMixin, forms.Select):
     pass
+
+
+class Select2ItemVarQuotaMulti(Select2Mixin, forms.SelectMultiple):
```

Comment:
```suggestion
class Select2ItemVarMulti(Select2Mixin, forms.SelectMultiple):
```
We have a item/var selection, not an item/var/quota selection, so that seems more appropriate?